pprofile.py

```diff
@@ -883,14 +962,37 @@ def runpath(path, argv, filename=None, threads=True, verbose=False):
     """
     _run(threads, verbose, 'runpath', filename, path, argv)
 
-_allsep = os.sep + (os.altsep or '')
+def _qCacheGringAdapt(path):
+    '''
+    qCacheGrind (windows build) needs at least one UNIX separator
+    in path to find the file. Adapt here even if this is probably
+    more of a qCacheGrind issue...
+    '''
+
+    if os.path.sep != "/":
```

Comment:
I think this can be expected to be invariant over interpreter process lifetime, so this test could be actually around the function definition instead of inside, making it computed once on module import rather than once per call:
```python
if os.path.sep != "/":
  def _qCacheGrindAdapt(path):
    ...
else:
  _qCacheGrindAdapt = lambda x: x
```